Add traffic-light instructions

jquack opened the pull request.

jquack committed:
Add traffic-light instructions

liampaull commented:
@miss01010001 can you take a look at the failed tests and try to fix - would love to have these instructions in the book asap.

tanij commented:
@liampaull all these seem like cross book reference issues. Shall we remove the references or merge with broken links?

jquack committed:
Replace corss-references with plain text

jquack committed:
Replace cross-references with plain text

jquack commented:
I removed the cross references and replaced them with plain text as a quick fix. :+1: 
However some parts of these instructions depend on other instruction sets, can someone help me on how to correctly include the cross-references?
